store/activities.js

```diff
@@ -2,31 +2,45 @@ const config = [
   {
     id: 'overview',
     path: 'overview',
+    icon: 'dashboard',
   },
   {
     id: 'content',
     path: 'content',
+    icon: 'document',
   },
   {
     id: 'depositDates',
     path: 'deposit-dates',
+    icon: 'calendar-check',
   },
   {
     id: 'plugins',
     path: 'plugins',
-  },
-  {
-    id: 'plugins/discovery',
-    path: 'plugins/discovery',
-  },
-  {
-    id: 'plugins/recommender',
-    path: 'plugins/recommender',
+    icon: 'puzzle',
+    subPaths: [
+      {
+        id: 'plugins/discovery',
+        path: 'plugins/discovery',
+      },
+      {
+        id: 'plugins/recommender',
+        path: 'plugins/recommender',
+      },
+    ],
   },
 ]
 
 const configMap = new Map(config.map(item => [item.id, item]))
 
 config.get = (...args) => configMap.get(...args)
+config.find = (...args) =>
+  config
+    .reduce((acc, curr) => {
```

Comment:
Prefer `flatMap` here. I would even do `flatMap` once after creating the array.

Reply:
That's really cool function. Thanks